ubuntu22.04/nvidia-driver

```diff
@@ -480,6 +482,14 @@ _shutdown() {
 }
 
 _resolve_kernel_type() {
+
+  # For backwards compatibility with older GPU Operator versions, honor the OPEN_KERNEL_MODULES_ENABLED
+  # variable when KERNEL_MODULE_TYPE is not set.
+  if [ -n ${OPEN_KERNEL_MODULES_ENABLED} ] && [ -z "${KERNEL_MODULE_TYPE}" ]; then
```

Comment:
In L20 we default the KERNEL_MODULE_TYPE to auto. Are we sure that this conditional will evaluate to true ?

Reply:
It won't, you are right. What about the following --  
1) do not set a default value for KERNEL_MODULE_TYPE in the driver container
2) GPU Operator 24.12.0+ should will always set the KERNEL_MODULE_TYPE envvar. Even if driver.kernelModuleType is not set in ClusterPolicy, the operator will set the envvar to `auto`.

Reply:
Yeah that sounds good to me